Kickstarter-iOS/Features/Login/Controller/LoginViewController.swift

```diff
@@ -70,17 +71,29 @@ internal final class LoginViewController: UIViewController {
   }
 
   override func bindStyles() {
-    _ = self |> loginControllerStyle
+    if featureDarkModeEnabled() {
+      _ = self.navigationController?.navigationBar.tintColor = adaptiveColor(.iconPrimary)
+    }
+
+    _ = self
+      |> loginControllerStyle
+
+    if featureDarkModeEnabled() {
```

Comment:
I think this is going to be a pain to clean up and I'm not sure how much value the flag is adding when the app is in forced light mode. I think we should at least consider not flag guarding these changes; happy to discuss this during a meeting if that's easier!

Reply:
It should be easy enough to add this attribute to the email and password text fields later, but I hear you. We need to flag-guard these. Otherwise, the adaptive colors are applied immediately and the idea is to fully rely on xcasset colors only once we're ready to roll out the full integration.